src/js/control-bar/seek-to-live.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * @file live-display.js
+ */
+import Button from '../button';
+import Component from '../component';
+import * as Dom from '../utils/dom.js';
+
+/**
+ * Displays the live indicator when duration is Infinity.
+ *
+ * @extends Component
+ */
+class SeekToLive extends Button {
+
+  /**
+   * Creates an instance of this class.
+   *
+   * @param {Player} player
+   *        The `Player` that this class should be attached to.
+   *
+   * @param {Object} [options]
+   *        The key/value store of player options.
+   */
+  constructor(player, options) {
+    super(player, options);
+
+    this.updateLiveEdgeStatus();
+    this.on(this.player_.liveTracker, 'liveedgechange', this.updateLiveEdgeStatus);
+  }
+
+  /**
+   * Create the `Component`'s DOM element
+   *
+   * @return {Element}
+   *         The element that was created.
+   */
+  createEl() {
+    const el = super.createEl('button', {
+      className: 'vjs-seek-to-live-control vjs-control'
+    });
+
+    this.contentEl_ = Dom.createEl('div', {
+      className: 'vjs-seek-to-live',
+      innerHTML: `<span class="vjs-control-text">${this.localize('Stream Type')}\u00a0</span>${this.localize('LIVE')}`
+    }, {
+      'aria-live': 'off'
+    });
+
+    this.circleEl_ = Dom.createEl('span', {
```

Comment:
@OwenEdwards this span creates a visual indicator that we're at the live edge. For accessibility, we'd probably want some control text here that says something like `at live edge`/`not at live edge`. Thoughts about wording or what type of info should be included?

Reply:
`updateLiveEdgeStatus` should then update the control text for this.